mcpgrafana.go

```diff
@@ -47,6 +48,7 @@ var ExtractGrafanaInfoFromEnv server.StdioContextFunc = func(ctx context.Context
 	if u == "" {
 		u = defaultGrafanaURL
 	}
+	slog.Info("Using Grafana configuration", "url", u, "api_key_set", apiKey != "")
```

Comment:
URL can contain a password, if we want to log the URL we should parse it and use `u.Redacted()`.